cypress/integration/entitybrowser_spec.js

```diff
@@ -1,23 +1,75 @@
+import helpers from '../helpers/helpers'
+
 describe('Entity Browser', function() {
   before(() => {
     cy.login()
   })
 
   beforeEach(() => {
-    cy.visit(`${getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
+    cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
+  })
+
+  describe('ListView', () => {
+    it('should load and display essential parts of the list view', function() {
+      cy.contains('Daten werden geladen...', {timeout: 8000})
+      cy.get('[title="Vorname"]')
+      cy.get('[type="submit"]')
+      cy.contains('Einträgen')
+      cy.contains('Aktionen auf Alle')
+      cy.get('[data-list-cell]').should('have.length.above', 10)
+    })
   })
 
-  const convert = s => s.replace(/ /g, '-').toLowerCase()
+  describe('DetailView', () => {
+    beforeEach(() => {
+      cy.get('[data-list-cell]').first()
+        .click()
+    })
+
+    it('should display detail view', function() {
+      cy.get('[data-cy=cypress-form-field]')
+        .should('have.length.above', 50)
+    })
+
+    it('should change value in detail view and save', function() {
+      cy.get('input#input-detailForm-callname')
+        .type('{selectall}{del}Test Callname')
+        .should('have.value', 'Test Callname')
+      cy.contains('Speichern')
+        .click()
+      cy.contains('Zuletzt gespeichert')
+    })
+
+    it('should change page back to list view without warning', function() {
+      cy.get('button').first()
+        .click()
+      cy.get('[data-list-cell]').should('have.length.above', 10)
+    })
+
+    it('should display warning on changing page back to list view on edited form', function() {
+      cy.get('input#input-detailForm-callname')
+        .type('{selectall}{del}')
+      cy.get('button').first()
```

Comment:
` cy.get('button').first()`
is very unstable. Please replace all such selectors